cola/widgets/remote.py

```diff
@@ -680,9 +689,45 @@ def push_to_all(self, _remote, *args, **kwargs):
         """Push to all selected remotes"""
         selected_remotes = self.selected_remotes
         all_results = None
+
+        pushed_remotes = 0
+
         for remote in selected_remotes:
             result = self.model.push(remote, *args, **kwargs)
+
+            if result[0] == 0:
+                pushed_remotes = pushed_remotes + 1
+
             all_results = combine(result, all_results)
+
+        if prefs.notify_on_push(self.context):
+            try:
+                notification = Notify()
+
+                total_remotes = len(selected_remotes)
+
+                if pushed_remotes == total_remotes:
+                    notification.title = N_('Push Successful')
+                    notification.icon = resources.package_data('icons', 'git-cola.svg')
+                elif pushed_remotes != 0:
+                    notification.title = N_('Push Partially Successful')
+                    notification.icon = resources.package_data(
+                        'icons', 'git-cola-error.svg'
+                    )
+                else:
+                    notification.title = N_('Push Failed')
+                    notification.icon = resources.package_data(
+                        'icons', 'git-cola-error.svg'
+                    )
+
+                notification.message = N_(
+                    f'{pushed_remotes}/{total_remotes} remotes updated'
+                )
+
+                notification.send()
+            except (ImportError, ModuleNotFoundError):
+                log.show_remote_messages(self, N_('notifypy dependency not found'))
```

Comment:
We don't need to emit a message that the notifypy dependency cannot be found. If it's there it's there, if it's not it's not.